src/stratis/run.rs

```diff
@@ -90,11 +94,15 @@ pub fn run(sim: bool) -> StratisResult<()> {
     runtime.block_on(async move {
         async fn start_threads<E>(engine: Arc<E>, sim: bool) -> StratisResult<()> where E: 'static + Engine {
             let (trigger, should_exit) = channel(1);
+
+            #[allow(unused_variables)]
```

Comment:
should we `#[cfg()]` flag this instead?

Reply:
Done. (The parser won't allow a flag inside a destructuring let, so it's a bit verbose, but still better, I think.)